plyer/facades/maps.py

```diff
@@ -0,0 +1,83 @@
+'''
+Maps
+=======
+The :class:`Maps` creates a client for accessing the default Maps API.
+
+Holds features such as opening a location by
+address & latitude/longitude, create queries, or find directions between
+two points
+
+Simple Examples
+---------------
+
+Perform a search::
+
+    >>> from plyer import maps
+    >>> maps.search('Mexican Restaurant')
+    >>> maps.search('Taco Bell', latitude=38.5810606, longitude=-121.493895)
```

Comment:
Better to avoid using a real company trade name here in our docs.

Reply:
Great point, changed.